torch_xla/csrc/ops/ops_xla_shape_fn.cpp

```diff
@@ -193,6 +193,25 @@ xla::Shape BinaryCrossEntropyBackwardOutputShape(
   return InferOutputShape(shapes, lower_for_shape_fn);
 }
 
+xla::Shape BitwiseAndTensorOutputShape(const torch::lazy::Value& input,
+                                       const torch::lazy::Value& other) {
+  return GetXlaShape(input);
```

Comment:
This doesn't seem right, for example
```
torch.bitwise_and(torch.tensor([-1], dtype=torch.int8), torch.tensor([1, 0, 3], dtype=torch.int8))

tensor([1, 0, 3], dtype=torch.int8)
```
You want to reuse lowering logic above(maybe make them helper functions).

Reply:
Done